composer.json

```diff
@@ -9,10 +9,16 @@
       "email": "matej.badal@profesia.sk"
     }
   ],
+  "repositories": [
```

Comment:
What is the reason behind this? Messaging core is registered in packagist, there shouldn`t be any need for such repository registration

Reply:
Yeah, it was just for testing purposes, and you're right, this kind of configuration is not needed.